operator_ui/src/createStore.ts

```diff
@@ -0,0 +1,18 @@
+import { createStore, applyMiddleware, Middleware } from 'redux'
+import { composeWithDevTools } from 'redux-devtools-extension'
+import thunkMiddleware from 'redux-thunk'
+import logger from 'redux-logger'
+import { createExplorerConnectionMiddleware } from './middleware'
+import reducer from './reducers'
+
+let middleware: Middleware[] = [
+  thunkMiddleware,
+  createExplorerConnectionMiddleware(),
+]
+if (process.env.LOG_REDUX === 'true') {
```

Comment:
This could get annoying since people often supply `TRUE`, lets do a case-insensitive match